recipe/meta.yaml

```diff
@@ -8,10 +8,12 @@ package:
 source:
     fn: Simbody-{{ version }}.tar.gz
     url: https://github.com/simbody/simbody/archive/Simbody-{{ version }}.tar.gz
-    sha256: 449c36e574d6f859d4fa8854ab6bc8e402e5ca5894bcce3e9fdce2f5658d64de
+    sha256: 7716d6ea20b950e71e8535faa4353ac89716c03fd7a445dd802eb6a630796639
 
 build:
-    number: 206
+    number: 1001
+    run_exports:
+        - {{ pin_subpackage('simbody', max_pin='x.x') }}
```

Comment:
I still don't quite understand this after reading the conda documentation but I guess it will only matter once someone trys to link against it in an upstream package. It can be adjusted then if needed.

Reply:
My understanding isn't great either.